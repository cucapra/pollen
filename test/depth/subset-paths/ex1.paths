../../subset-paths/ex1.paths