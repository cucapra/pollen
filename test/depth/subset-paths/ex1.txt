../../subset-paths/ex1.txt