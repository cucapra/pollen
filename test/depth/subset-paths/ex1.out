#node.id	depth	depth.uniq
1	1	1
2	2	1
